src/tests/Makefile

```diff
@@ -3,6 +3,10 @@
 ALL_TEST_CXXFLAGS=$(CPPFLAGS) $(TEST_CXXFLAGS) $(TEST_CXXFLAGS_EXTRA)
 ALL_TEST_LDFLAGS=$(TEST_LDFLAGS) $(TEST_LDFLAGS_EXTRA)
 
+ifeq ($(SUPPORT_CAPABILITIES),1)
+  ALL_TEST_LDFLAGS+=-lcap
+endif
```

Comment:
This isn't really the right place for this. If tests need `-lcap` then it should be in `TEST_LDFLAGS`. That could be set in `mconfig.Linux`/`mconfig.Linux.sh`, but see comment there.

Reply:
all linking needs -lcap due to the IAB abstraction in the load-service.h

Reply:
It can go into both `TEST_LDFLAGS` and `LDFLAGS`, what I mean is that this should happen in the build configuration (mconfig.* / configure script) though rather than being done ad-hoc in various makefiles.